Installation.md

```diff
@@ -8,6 +8,14 @@ You may need to install them manually. They are:
 * bash 4.0+
 * Git 2.17+
 * `column`
+* `awk`
+* `column`
+* cURL
+* `find`
+* `ps`
+* `rsync`
```

Comment:
Let's doc what commands are using rsync. There are few commands that are using rsync, if the user doesn't need them, she doesn't need rsync too.

Reply:
Sounds good, I did that for a few more commands :+1: